src/meta/service/src/ddl_service.rs

```diff
@@ -428,6 +430,21 @@ impl DdlService for DdlServiceImpl {
             // Generate source id.
             let source_id = self.gen_unique_id::<{ IdCategory::Table }>().await?; // TODO: Use source category
             fill_table_source(source, source_id, &mut mview, table_id, &mut fragment_graph);
+
+            // Modify properties for cdc sources if needed
+            if let Some(connector) = source.properties.get(UPSTREAM_SOURCE_KEY) {
+                if matches!(
+                    CdcSourceType::from(connector.as_str()),
+                    CdcSourceType::Mysql
+                ) {
+                    let server_id = self
+                        .env
+                        .id_gen_manager()
+                        .generate::<{ IdCategory::MySqlCdc }>()
+                        .await?;
```

Comment:
IIUC, the server id needs to be unique in MySQL, not in RW. Therefore, I feel that it can be easy to generate conflicts by using RW's  id_gen_manager here since it generates a monotonically increasing ID. Also, RW's id_gen_manager will persist the next_allocate_id in etcd, which seems unnecessary in this case.

How about just generating a random id for mysql server.id?

Reply:
Yes, it should be unique. random id also sounds good to me.

Reply:
Plz use a random id for it since we gonna deprecate `id_gen_manager` after sql-backend introduced.

Reply:
done. PTAL again